public/index.html

```diff
@@ -0,0 +1,63 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width,initial-scale=1">
+  <title>Space work reviews</title>
+  <style>
+    .container {
+      margin: auto;
+      width: 90%;
+      max-width: 1200px;
+      display: grid;
+      grid-template-columns: 60% 40%;
+      grid-gap: 15px;
+    }
+
+    @media screen and (max-width: 1000px) {
+      .container {
+        display: block;
+      }
+
+      .col-2 {
+        display: none;
+      }
+    }
+
+    .col-1 {
+      /* background: blue; */
+      height: auto;
+    }
+
+    .col-2 {
+      /* background: red; */
+      height: 100vh
+    }
+  </style>
+  <!-- <link rel="stylesheet" href="/reset.css"> -->
+  <script defer="defer" src="/reviews.js"></script>
+  <link href="/main.css" rel="stylesheet">
+</head>
+
+<body>
+  <div id="navbar"></div>
+  <div class="container">
+    <div class="col-1">
+      <div id="reviews"></div>
+    </div>
+    <div class="col-2">
+      <div id="app"></div>
+    </div>
+  </div>
+  <div id="footer"></div>
+</body>
+<script crossorigin src="https://unpkg.com/react@17/umd/react.production.min.js"></script>
+<script crossorigin src="https://unpkg.com/react-dom@17/umd/react-dom.production.min.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/axios/0.21.1/axios.min.js"
```

Comment:
nice dude! are you gonna use CDN's for all of your npm packages in the end?

Reply:
Yes, it is looking that way